key_files/globalmeantemperature.yml

```diff
@@ -0,0 +1,22 @@
+---
+name            : GlobalMeanTemperature
+units           : degrees C
+dimensions      : 1
+model           : NEMO
+modelFiles      : $BASEDIR_MODEL/$JOBID/nemo_$JOBIDo_1y_*_grid-T.nc
+gridFile        : $PATHS_GRIDFILE                                                                                      
+model_vars      : thetao_con
+model_t         : time_centered         # model time dimension
+model_cal       : 360_day               # model calendar
+model_z         : deptht                # model depth dimension
+model_lat       : nav_lat               # model latitude dimension
+model_lon       : nav_lon               # model latitude dimension
+model_convert   : bgcval2/functions/globalVolMean.py:globalVolumeMean
+model_convert_areafile : $PATHS_GRIDFILE
+layers          : layerless
+regions         : regionless
+
+# changes from original:
+#   add --- at start
+#   remove [name]
+#   add bgcval2 berfore functions 
```

Comment:
what with the commented out bits?

Reply:
They're comments! This is what I've had to do to bgc-val-public code to fit it into bgcval2.

Reply:
ah OK, makes sense then